agent/doc/swdesign/README.md

```diff
@@ -671,6 +672,23 @@ Needs:
 - impl
 - utest
 
+##### RuntimeFacade deletes old workload without sending workload states
+`swdd~agent-delete-old-workload-without-sending-workload-states~1`
+
+Status: approved
+
+When the RuntimeFacade is requested to delete the workload and the flag `report_workload_states_for_workload` is false, then the RuntimeFacade shall delete the workload via the runtime connector without sending workload states.
+
+Comment:
+This delete is done by the specific runtime for a workload Id. No internal workload object is involved in this action.
+
```

Comment:
Maybe we can add a rationale stating that this is a workaround needed until a dedicated workload restart handling is implemented.

Reply:
I have added a rationale.

Reply:
Fixed